### DISCUSSION TITLE
Any insights on how Google Scholar indexes research from Biblio?

### DISCUSSION THREAD

**Discussion by mietcls** [ugent-library/biblio-backend#1352] [Q&A] [answered]
A researcher told us an old chapter is being indexed by Google Scholar, that was not meant to be indexed.

# Google indexation

This is an old title of a chapter apparently, that shows a link to the (current, updated) Biblio full text: https://scholar.google.com/scholar?hl=nl&as_sdt=0%2C5&q=Consistently+low+chytrid+prevalence+throughout+tadpole+development+in+a+recovering+frog+species&btnG=

This is the actual full text:
https://scholar.google.com/citations?view_op=view_citation&hl=nl&user=LVtTSZgAAAAJ&sortby=pubdate&citation_for_view=LVtTSZgAAAAJ:P5F9QuxV20EC

Both are shown on the researcher's profile: https://scholar.google.com/citations?hl=nl&user=LVtTSZgAAAAJ&view_op=list_works&sortby=pubdate

# Question
Can we see whether this record used to have a different name, and this might have gotten indexed? Is Google itself doing something that causes these results to appear?
How does it work?

## Biblio context
- Record: 01HHYJCBD46TBFCF0FJR1VCAZE
- Back Office link: https://backoffice.biblio.ugent.be/publication/01HHYJCBD46TBFCF0FJR1VCAZE?redirect-url=%2Fpublication%3Fq%3DAmphibian%2Bpersistence%2Bin%2Bthe%2BAnthropocene%26sort%3Ddate-updated-desc
- Front office link: https://biblio.ugent.be/publication/01HHYJCBD46TBFCF0FJR1VCAZE

**Comment by nics** [comment, marked as answer]
That's all complete voodoo, we have very little insight. We make sure different versions of the same record have different timestamps etc but junk still gets in their index. We can request a reindex of a page manually though https://support.google.com/webmasters/answer/9012289#request_indexing. With any luck this then percolates into Google Scholar?

**Reply by mietcls** [reply]
Thank you, that's interesting! It's worth the try. However, how much work is this (I'm already stuck at step 1), have we done this before? Doing this manually will probably not scale and it might not even work. I can't figure out from their guidelines what might be wrong on our end.

**Reply by nicolasfranck** [reply]
I would help us the see the effects of our work. Maybe a record hasn't been indexed in a long time because it could not reach the record (due to throttling, or downtimes ..)? Doing this one by one reduces that risk

**Reply by mietcls** [reply]
Solved: I now have access and can request reindexing for exceptional cases such as this one. Because our front office was down, this happened to a couple of records I noticed.
